Singer.API/ClientApp/src/app/modules/shared/components/detailed-user-card/detailed-user-card.component.html

```diff
@@ -0,0 +1,60 @@
+<mat-card class="card">
+   <mat-card-content>
+      <div class="card-content">
+
+         <div class="head">
+            <div class="head__initials" [ngStyle]="{'background': color}">
```

Comment:
Why do we have class names with double underscores?

Reply:
Because I Implemented the BEM (Block Element Modifier) Naming convention for our CSS to make it more readable and more organized.

Reply:
example: 
class="head__initials color--white"
class="Block__Element Modifier"

Reply:
OK, I think it's a good idea in itself, but I don't like that we now have 2 naming conventions across this project in our CSS files. Please create a task to clean this up in the next sprint

Reply:
Right, this answers my question too.